### PR TITLE
[docs] Move YCQL Build an app pages to Drivers and ORMs

### PR DESCRIPTION
**Pull request by aishwarya24**
Description:
1. Moved YCQL pages from Java, Go, C#, Python, Node.js. 
2. Added YSQL and YCQL tabs and sorted the drivers based on them.
3. Redirects added in netlify.toml for the deleted YCQL files.
4. Added the table entries(incomplete at the moment) for the YCQL drivers in the drivers and ORMs index files.

### CODE REVIEW

File: docs/content/preview/drivers-orms/java/datastax-ycql-4.6.md [hunk 1]
```diff
@@ -1,82 +1,34 @@
 ---
-title: Build a Java application that uses YCQL
-headerTitle: Build a Java application
-linkTitle: More examples
-description: Build a sample Java application with the Yugabyte Java Driver for YCQL v4.6.
+title: Connect an application
+linkTitle: Connect an app using YCQL
+description: JDBC drivers for YCQL
```
Comment (ashetkar) [inline docs/content/preview/drivers-orms/java/datastax-ycql-4.6.md:4]: It's not a JDBC Driver, but a Java driver for YCQL.

File: docs/content/preview/drivers-orms/java/datastax-ycql-4.6.md [hunk 2]
```diff
@@ -1,82 +1,34 @@
...
-      YSQL - JDBC SSL/TLS
-    </a>
-  </li>
   <li >
-    <a href="../ysql-hibernate/" class="nav-link">
-      <i class="icon-postgres" aria-hidden="true"></i>
-      YSQL - Hibernate
+    <a href="../datastax-ycql/" class="nav-link">
+      <i class="icon-java-bold" aria-hidden="true"></i>
+      YugabyteDB YCQL DataStax Driver
```
Comment (ashetkar) [inline docs/content/preview/drivers-orms/java/datastax-ycql-4.6.md:18]: Don't need to use Datastax name in the title, IMO.

**Review by ddhodge — APPROVED** [review]
misc comments

File: docs/content/preview/drivers-orms/java/yugabyte-jdbc.md [hunk 3]
```diff
@@ -11,11 +11,26 @@ menu:
...
+
 <ul class="nav nav-tabs-alt nav-tabs-yb">
 
   <li >
     <a href="../yugabyte-jdbc/" class="nav-link active">
-      <i class="icon-java-bold" aria-hidden="true"></i>
+      <i class="icon-postgres" aria-hidden="true"></i>
       YugabyteDB JDBC Smart Driver
     </a>
   </li>
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/yugabyte-jdbc.md:36]: In parameters table below, I think it should be topology-keys not topology_keys

File: docs/netlify.toml [hunk 4]
```diff
@@ -398,6 +398,33 @@
   to = "/preview/quick-start/#create-a-local-cluster"
   force = true
 
+# Build an application redirects to Drivers and ORMs
+
+[[redirects]]
```
[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/netlify.toml:403]: In general, we should prefer redirects for reasons of SEO and maintainability.

I also suspect we could wildcard these, but I haven't actually checked to make sure nothing else would match this:

```
from = "/preview/develop/build-apps/:lang/:query"
to = "/preview/drivers-orms/:lang/:query"
```

**Comment by nchandrappa** [comment]
LGTM.

File: docs/netlify.toml (hunk 4 above)
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/netlify.toml:403]: @polarweasel , Ill try to add this in my next PR. Thanks.

**Pull request merged**